src/calendar/calendar.component.ts

```diff
@@ -41,7 +41,7 @@ export class CalendarComponent implements OnDestroy {
         this.generateMonthData();
     }
 
-    initCalendar(date: moment.Moment, minDate: moment.Moment, maxDate: moment.Moment) {
+    initCalendar(date: moment.Moment, minDate: moment.Moment = this.minDate, maxDate: moment.Moment = this.maxDate) {
```

Comment:
that's cool that you can use a this variable as the default... I suppose when it transpiles it would be more traditional.